ui/imports/shared/controls/chat/UserImage.qml

```diff
@@ -18,7 +18,6 @@ Loader {
     property string image
     property bool showRing: true
     property bool interactive: true
-    property var messageContextMenu
```

Comment:
:+1: that was not the place for `messageContextMenu` as `UserImage` is also used in different contexts than the message view, thanks for removing it from here